README

```diff
@@ -83,6 +100,17 @@ export USERLAND_ARCHIVES
 
         $ cd (component-dir) ; gmake publish
 
+    If you build several components, and some are required dependencies for the
```

Comment:
ok, i'll remove it (tomorrow probably) and tackle the ability to depend on
local just-built repo separately.

I thought encumbered should be good (didn't break anything for me when
applied to profile after 'gmake setup') - but seems there should appear an
equivalent constrained to compilation or even publishing of just the leaf
components?

--
Typos courtesy of GMail on my Samsung Android
05.04.2016 21:12 пользователь "Alexander Pyhalov" <notifications@github.com>
написал:

> In README
> <https://github.com/OpenIndiana/oi-userland/pull/1806#discussion_r58598126>
> :
>
> > @@ -83,6 +100,17 @@ export USERLAND_ARCHIVES
> >
> >          $ cd (component-dir) ; gmake publish
> >
> > +    If you build several components, and some are required dependencies for the
>
> Sorry, haven't seen this. We removed references to ENCUMBERED as setting
> ENCUMBERED in ~/.profile breaks gmake components.mk in components dir.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenIndiana/oi-userland/pull/1806/files/a5cf2dfed8ea794180a37e12dfbb69d63d7f7f98#r58598126>
>